buildlib/azure-pipelines.yml

```diff
@@ -36,25 +34,21 @@ stages:
               set -x
               check_release_build $(Build.Reason) $(Build.SourceVersion) "AZP/SNAPSHOT: "
             name: Result
+            
   - stage: Build
     dependsOn: Prepare
     condition: eq(dependencies.Prepare.outputs['Check.Result.Launch'], 'True')
     jobs:
       - template: jucx/jucx-build.yml
         parameters:
           arch: amd64
-          container: centos7_cuda11_x86_64
+          container: centos8_cuda11_x86_64
           demands: ucx_docker
+          target: publish-snapshot
 
       - template: jucx/jucx-build.yml
         parameters:
           arch: aarch64
           container: centos8_cuda11_aarch64
           demands: ucx-arm64
-          
-      - template: jucx/jucx-publish.yml
```

Comment:
do we still use the file jucx-publish.yml? can we remove it?

Reply:
jucx-publish.yml is still used for Maven Central publishing.